src/storage/src/hummock/state_store.rs

```diff
@@ -107,7 +111,7 @@ impl HummockStorage {
 
         // Generate iterators for versioned ssts by filter out ssts that do not overlap with given
         // `key_range`
-        for level in pinned_version.levels() {
+        for level in pinned_version.levels(compaction_group_id) {
```

Comment:
In the current implementation, read and write interfaces are bound to a compaction group, which assumes that the mapping between table_id and compaction_group is static and correct. 

I think it is better to leave comments for this assumption since we will need to revisit and refactor all the changes if we want to support dynamic compaction group.